pkg/operator/ceph/object/spec.go

```diff
@@ -154,11 +154,24 @@ func (c *clusterConfig) makeRGWPodSpec(rgwConfig *rgwConfig) (v1.PodTemplateSpec
 		ServiceAccountName: serviceAccountName,
 	}
 
+	rgwOpsLogEnabled := false
+	if opsLogSidecar := c.store.Spec.Gateway.OpsLogSidecar; opsLogSidecar != nil {
+		rgwOpsLogEnabled = true
+		// Generate ops log file name based on rgw daemon ID
+		opsLogFile := fmt.Sprintf("ops-log-%s", getDaemonName(rgwConfig))
```

Comment:
What does `getDaemonName(rgwConfig)` render out to? I want to make sure that somewhere in the `ops-log-*.log` filename, the keyword `rgw` is guaranteed to be present to help understand log file context from filename. If this doesn't render an `rgw` keyword in all cases, let's make this `rgw-ops-log-%s.log`.

Reply:
It will have rgw in the name in the format "ops-log-rgw-client-a.log" which coincides perfectly with the default file generated by rgw